src/sentry/api/endpoints/monitor_checkins.py

```diff
@@ -137,9 +146,22 @@ def post(
         result = serializer.validated_data
 
         with transaction.atomic():
+            environment = Environment.get_or_create(
+                project=project, name=result.get("environment", "production")
+            )
+
+            monitor_environment, created = MonitorEnvironment.objects.get_or_create(
+                monitor=monitor, environment=environment
+            )
+
+            if created:
+                monitor_environment.status = monitor.status
```

Comment:
Should we copy over anything else at creation time like this?

Reply:
Yeah, I guess we can copy over the `next_checkin` and `last_checkin` but we're going to update it below regardless